cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala

```diff
@@ -502,6 +503,40 @@ class SchemaIndexTest extends DocumentingTestBase with QueryStatisticsTestSuppor
     )
   }
 
+  @Test def use_node_token_lookup_index() {
+    profileQuery(
+      title = "Node label LOOKUP index example",
+      text = "In the example below, a node TOKEN LOOKUP index is available.",
+      prepare = Some(_ => executePreparationQueries(List(
+        "CREATE LOOKUP INDEX node_label_lookup_index FOR (n) ON EACH labels(n)",
```

Comment:
For the CREATE LOOKUP examples we do 
```
// remove autogenerated token lookup indexes
val res = execute("SHOW LOOKUP INDEXES YIELD name")
res.columnAs[String]("name").foreach(n => execute(s"DROP INDEX $n"))
```
I see you are not failing the build on this so it might not be needed but just a thought (relevant for both the node and relationship example)